Marlin/Configuration.h

```diff
@@ -1,4 +1,4 @@
-/**
+﻿/**
```

Comment:
Has the encoding been changed on this or a BOM added?

Reply:
I think my editor did it (added BOM). To properly display Prusa I have to save it in utf8..I didn't revert code using git but only restoring content and this may be the result.